src/publicapi.js

```diff
@@ -158,6 +158,10 @@ function getInterface(v) {
       if (this.__controller.blurred) this.__controller.cursor.hide().parent.blur();
       return this;
     };
+    _.empty = _.clear = function() {
+      this.select().__controller.backspace();
+      return this;
+    };
```

Comment:
Yeah, the `.select()` thing isn't ideal - we must come up with a better approach.

@karoltarasiuk was there a reason for the double `.empty()` / `.clear()` naming? We only need one method name on our side.

Just to add some context @laughinghan: we added this method so we could clear just the currently focused `\MathQuillMathField{}` without affecting stuff around it. If there were no `\MathQuillMathField{}`s, we wouldn't need this method as we could just call `.latex('')`.